etc/profile-m-z/skypeforlinux.profile

```diff
@@ -6,7 +6,6 @@ include skypeforlinux.local
 include globals.local
 
 # Disabled until someone reported positive feedback
-ignore whitelist ${DOWNLOADS}
 ignore include whitelist-common.inc
```

Comment:
We should remove this too.

Reply:
`whitelist-common.inc` grants write access to `.config/dconf`, and this sounds like it could be abused to run arbitrary commands (for example, by changing the default terminal).

Reply:
1. https://github.com/netblue30/firejail/blob/facf03c54108ff7180339f09425c2b3fdc651efd/etc/inc/disable-common.inc#L155
2. Who cares about such hacks for a profile which allows `systemd-run` & co.